ale_linters/make/checkmake.vim

```diff
@@ -0,0 +1,24 @@
+" Author: aurieh - https://github.com/aurieh
+
+function! ale_linters#make#checkmake#Handle(buffer, lines) abort
```

Comment:
You might be able to use the standard Unix error handler here instead. `ale#handlers#unix#HandleAsError` might work here.

Reply:
Standard Unix error handler regex doesn' seem to match. I could probably modify checkmake's `--format` flag to make it work.

Reply:
Actually, I take that back: I've tried multiple format string combinations, still can't get the Unix handler to work.

Reply:
You could either make it match, or write add Vader tests for the new function. Either works for me.

Reply:
Maybe do the latter, keeping this in mind: #852

Reply:
I'll try to write a Vader test. Should I just pass a static input string or?